eblob_kit.py

```diff
@@ -559,6 +623,25 @@ def data(self):
         """Return data file."""
         return self._data_file
 
+    def get_index_data_path_tuple(self):
+        """Get underlying files paths."""
+        return self.index.path, self.data.path
+
+    def close(self):
+        """Closes underlying files."""
+        self._index_file.close()
+        self._data_file.close()
+
+    @staticmethod
+    def move_back(from_index, from_data, to_index, to_data):
```

Comment:
There is also could be a `.data_is_sorted` file that should be either left or removed.

Reply:
Fixed: added processing of data blobs marked with ```data_is_sorted```.